services/debugger/uploader_test.go

```diff
@@ -298,3 +293,12 @@ var _ = Describe("Uploader", func() {
 		})
 	})
 })
+
+func assertRequest(req *http.Request) {
+	username, password, ok := req.BasicAuth()
+	Expect(ok).To(BeTrue())
+	Expect(username).To(Equal("test"))
+	Expect(password).To(Equal("test"))
```

Comment:
It's not a good practice to pass the same value to multiple fields.

If the username/password got inverted you won't be able to detect this